src/v3/test/integration/__snapshots__/authenticator-enroll-select-authenticator.test.tsx.snap

```diff
@@ -516,7 +517,7 @@ exports[`authenticator-enroll-select-authenticator renders form 1`] = `
                     <div
                       class="ods-4o5zYQ ods-2oc5SK ods-4XAbls ods-2NLKpK ods-1ll9ky ods-4IdjLs description"
                     >
-                      Verify by entering a temporary code from the {0} app.
```

Comment:
Another fix, translation key was not passing proper param value but now using loc function it is correct.